src/main/resources/xml/schema/2020-1/nordic2020-1.sch

```diff
@@ -784,7 +779,7 @@
     <pattern id="epub_nordic_269">
         <title>Rule 269</title>
         <p></p>
-        <rule context="html:body[not(html:header)]">
+        <rule context="html:body/html:section[not(html:header)]">

```

Comment:
Note that checking for the header elment is not necessary any more. The `<header>` was used in the 2015-1 guidelines as a wrapper around the HTML elements corresponding to [the DTBook `<doctitle>`, `<docauthor>` and `covertitle` elements](https://github.com/daisy/pipeline/blob/f65e446baa8764d85b5a4f26a3e88dff5e786191/modules/scripts-utils/dtbook-utils/src/main/resources/xml/dtd/dtbook-2005-3.dtd#L2209-L2241). The `<header>` element was the only allowed element other than `<section>` (and the never-used `<article>`), and was always always present in the HTML version (single HTML file), but never present in the EPUB version (multiple HTML files). There are several places in the schematrons where we check for the header element, and it is almost always used as a test for whether we're validating a single HTML file, or a HTML file which is part of an EPUB.

In this case, the context `html:body[not(html:header)]` matches `<body>` elements for when the book is split into multiple HTML files (EPUB) but not when the book is in a single HTML file (HTML version).

Reply:
I have removed it. Please check the update.